ext/storage_sources/azure_store/azure_connection.cpp

```diff
@@ -31,16 +31,35 @@
 #include <azure/core.hpp>
 #include <azure/storage/blobs.hpp>
 
+#include <stdlib.h>
```

Comment:
`#include <cstdlib>` is more idiomatic for C++.

Reply:
okies made the change :)